### DISCUSSION TITLE
Export to OFX

### DISCUSSION THREAD

**Discussion by vanillajonathan** [NickvisionApps/Denaro#652] [Ideas]
Denaro can import from a OFX file but not export.

Add a feature to export to a OFX file.

**Comment by nlogozzo** [comment]
I have to look into OFX but right now we only import transactions from OFX as I believe it doesn't support groups and tags. (QIF supports transactions and groups, but idk about tags)...therefore in not comfortable exporting to these formats and losing a large amount of account data

**Reply by vanillajonathan** [reply]
Denaro could inform the user upon export that not tags are not exported.

**Reply by nlogozzo** [reply]
> Denaro could inform the user upon export that not tags are not exported.

But neither will groups...so I don't really see the benefit

**Reply by vanillajonathan** [reply]
Yeah, it could mention that too. It should mention everything that isn't exported so that  the user is aware and can make an informed decision.

**Comment by fsobolev** [comment]
What is the use case for exporting Denaro data to OFX?

**Reply by vanillajonathan** [reply]
Importing data to other software which supports OFX import.

**Reply by fsobolev** [reply]
That's exactly the reason why this should not be implemented IMO. Denaro still has missing features and only runs on Linux but I think Nick will agree with me that Denaro is meant to be used as main money management app, not as companion app to Homebank/GNU Cash/something else. We should spend time on making it better to allow more people to mainly use Denaro instead of making it just an additional tool to use instead of other apps in some cases. So export to OFX or any other money management format contradicts the purpose of Denaro and doesn't worth spending time on it.

**Reply by vanillajonathan** [reply]
Then it could be a low priority issue, but I think it would be good to support export to formats so that it can be imported into other applications so that a user is not locked into Denaro (yes, I know there is CSV export).
